ROADMAP.md

```diff
@@ -0,0 +1,71 @@
+# Roadmap + Designs
+
+This document contains a rough outline of a roadmap and a few designs for future features in Apollo Server. Contributions are very welcome! Feel free to pick an of the upcoming features and start implementing, and don't hold back with questions or ideas!
+
+
+## Roadmap
+
+### Completed
+
+* Rewrite in TypeScript
+* Simplification of API
+* Express integration
+* Query batching (Express)
+* Query whitelisting / stored queries
+
+### In progress
+
+* HAPI integration
+
+* Complete rewrite of Apollo Server documentation
+
+### Next up
+
+* Koa integration
+* Connect integration
+* Better GraphQL error handling
+* Support for simple query timeouts
+* Websocket transport
+
+### Future
+
+* Support for @defer, @stream and @live directives
```

Comment:
My favorite thing about that is that I believe it can be done without changes to GraphQL-JS.